Doppler.Currency.Test/Integration/TestServerFixture.cs

```diff
@@ -17,19 +17,19 @@ public class TestServerFixture : IDisposable
 
         public HttpClient Client { get; }
 
-        public Mock<IBnaService> BnaServiceMock;
+        public Mock<ICurrencyService> CurrencyServiceMock;
         public Mock<ISlackHooksService> SlackHookServiceMock;
```

Comment:
Why are you making these fields and property public?

Reply:
Because I need access to this variable to setup methods of this object in the tests for example 
https://github.com/FromDoppler/doppler-currency/blob/master/Doppler.Currency.Test/Integration/UsdCurrencyControllerTest.cs#L21

Reply:
Oh, I understand now.

> Maybe can be protected internal

Nah, _public_ is fine, just convert it to properties:

```diff

-        public Mock<IBnaService> BnaServiceMock;
-        public Mock<ICurrencyService> CurrencyServiceMock;
-        public Mock<ISlackHooksService> SlackHookServiceMock;
+        public Mock<IBnaService> BnaServiceMock { get; }
+        public Mock<ICurrencyService> CurrencyServiceMock { get; }
+        public Mock<ISlackHooksService> SlackHookServiceMock { get; }
```

Reply:
Yes, Done